### ISSUE TITLE
getTimeStringSeparator needs consider '.'

### ISSUE THREAD

**Issue by kissge** [Bwca/iso-datestring-validator#33]
```js
> getTimeStringSeparator('025001.957375256+0900')
'.'
```

Due to this,

```js
> isValidISODateString('20220402T025001.957375256+0900')
false
```

**Comment by Bwca** [comment]
Good point, thanks for reporting, gonna fix it 🙂

**Comment by Bwca** [comment]
Published v 2.2.2, it should fix the issue :)

**Comment by kissge** [comment]
Great work 😍
